localStorage.js

```diff
@@ -0,0 +1,33 @@
+/* eslint-disable */
```

Comment:
[OPTIONAL] Please don't disable linters. If you check the error you will get why we need them. It helps us to have a consistent codebase even if there are many contributors to one codebase. Also, it helps us to make better choices. Check this [link](https://sourcelevel.io/blog/what-is-a-linter-and-why-your-team-should-use-it) to learn more. 
<img width="753" alt="Screenshot 2023-01-07 at 1 42 47 AM" src="https://user-images.githubusercontent.com/25887253/211112490-1a142a38-fd86-4ac9-837c-d456d28a40ed.png">